packages/mana/basevector.go

```diff
@@ -48,9 +44,9 @@ type BaseManaVector interface {
 func NewBaseManaVector(vectorType Type) (BaseManaVector, error) {
```

Comment:
Do we need this function now? Both cases in the switch perform the same action anyway

Reply:
True! I've turned it into a basic constructor.